# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../prelude-correlator/install_sysconfdir.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.am.orig	2007-08-12 01:24:54.000000000 +0200
+++ ./Makefile.am	2007-08-12 01:25:22.000000000 +0200
@@ -4,7 +4,8 @@
 
 
 install-data-local:
-	$(INSTALL) -m 700 -d $(prelude_correlator_context_dir)
+	$(INSTALL) -m 700 -d $(DESTDIR)$(configdir);
+	$(INSTALL) -m 700 -d $(prelude_correlator_context_dir);
 	@if test -f $(DESTDIR)$(configdir)/prelude-correlator.conf; then                                        \
 		echo "********************************************************************************";        \
 		echo;                                                                                           \
--- ./plugins/pcre/ruleset/Makefile.am.orig	2000-01-01 00:00:00.000000000 +0100
+++ ./plugins/pcre/ruleset/Makefile.am	2007-08-12 01:39:09.000000000 +0200
@@ -4,5 +4,5 @@
 		pcre.rules \
 		scan.rules \
 		worm.rules
-rulesetdir = $(PRELUDE_CORRELATOR_CONFIG_PREFIX)/ruleset
+rulesetdir = $(DESTDIR)$(configdir)/ruleset
 EXTRA_DIST = $(ruleset_DATA)
